go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/scrapli/scrapligocfg v1.0.0
 	github.com/spf13/cobra v1.6.1
 	github.com/spf13/pflag v1.0.5
-	github.com/srl-labs/srl-controller v0.4.6
+	github.com/srl-labs/srl-controller v0.4.7-0.20230311230716-e1c96973d463
```

Comment:
the container is v0.5.0 and the corresponding lib is v0.4.7, is that intended?

Reply:
yes, just for testing inside this PR.
If tests are ok I will take both as 0.5.0

I spoke to Marcus and I will regenerate the manifests to use `:latest` container tag, so that manifest stays more-or-less static

Reply:
yeah I tested on my machine, good to switch to 0.5.0

Reply:
done